test.js

```diff
@@ -1,11 +1,14 @@
+import os from 'os';
 import test from 'ava';
 
 test.serial('username()', async t => {
+	os.userInfo = () => ({username: 'unicorn'});
```

Comment:
Can you make a separate test for this? Without the `process.env` thing. Same with the sync version too.